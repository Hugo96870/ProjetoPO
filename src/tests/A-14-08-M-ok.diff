*** tests/A-14-08-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-14-08-M-ok.outhyp	2020-12-03 16:57:04.043633300 +0000
***************
*** 25,45 ****
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: BOX|caixa1|P2|1|1|0|AIR
  Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
--- 25,47 ----
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: BOX|caixa1|P3|12|12|0|NORMAL
! Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Registar Caixa: Operação inválida: O produto 'caixa1' já existe.
! Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: BOX|caixa1|P3|12|12|0|NORMAL
  Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
***************
*** 55,61 ****
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Titulo do livro: Autor do livro: ISBN: Preço: Valor crítico para as existências: Identificador do fornecedor: Registar Livro: Operação inválida: O fornecedor 'P100' não existe.
  Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
--- 57,63 ----
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Autor do livro: Titulo do livro: ISBN: Preço: Valor crítico para as existências: Identificador do fornecedor: Registar Livro: Operação inválida: O fornecedor 'P100' não existe.
  Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
