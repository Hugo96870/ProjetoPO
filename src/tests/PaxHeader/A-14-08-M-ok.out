30 mtime=1606841711.210019925
